make.mk

```diff
@@ -0,0 +1,12 @@
+# VERSION=1.0.0
+
+GITHUB_BASE ?= https://raw.githubusercontent.com/ccnmtl/makefiles/master/
+WGET ?= wget
+WGET_FLAGS ?= "-O"
+
+update_makefiles:
+	for f in $$(grep -l "^# VERSION=" *.mk) ; do \
+		$(WGET) $(GITHUB_BASE)$$f $(WGET_FLAGS) $$f ; \
+	done
+
+.PHONY: update_makefiles
```

Comment:
You can pick up the .eslintrc.js file from passhugo: https://github.com/ccnmtl/passhugo/blob/master/.eslintrc.js

Reply:
👍